manifests/m/MacType/MacType/2021.1-rc1/MacType.MacType.yaml

```diff
@@ -0,0 +1,17 @@
+PackageIdentifier: MacType.MacType
+Publisher: MacType
```

Comment:
```suggestion
Publisher: FlyingSnow, Samantha Glocker
```

Reply:
I think it is not necessary to modify this, just like `PicGo/PicGo`.

Reply:
Windows Package Manager relies on Control Panel's data and it has to match so others can use the `winget list` and `winget upgrade` commands.